services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet | Movie:
```

Comment:
The return type annotation `QuerySet | Movie` is incorrect because the function always returns a `QuerySet`. Consider changing it to `QuerySet`.